app/src/main/res/values/strings.xml

```diff
@@ -481,7 +481,8 @@
     <string name="restore_confirmation">Overwrite My Notes, Bookmarks, Reading Plans and Workspaces?</string>
     <string name="restore_success">My Notes, Bookmarks, Reading Plans and Workspaces restored successfully</string>
     <string name="loading_backup">Loading database from backup…</string>
-    <string name="backup_success">My Notes, Bookmarks, Reading Plans and Workspaces backed up successfully to %s.</string>
+    <string name="backup_success">My Notes, Bookmarks, Reading Plans and Workspaces backed up successfully.</string>
```

Comment:
Because of translations, we should not make such changes to strings that change parameters. Otherwise users with translations that have not been updated will have crashes.

When parameters need to be changed, like here, we need to introduce a new string (with new key, for example backup_success2), use it, and remove old string.